src/Public/Routes.ps1

```diff
@@ -173,8 +195,17 @@ function Add-PodeRoute {
         [switch]
         $Logout,
 
+        [hashtable]
+        $OAResponses,
+
+        [string]
+        $OAReference,
+
         [switch]
-        $PassThru
+        $PassThru,
+
+        [string[]]
+        $OADefinitionTag
```

Comment:
I think we should change the logic to 

 if ([string]::IsNullOrWhiteSpace($Path) -and ![string]::IsNullOrWhiteSpace($RouteGroup.Path)) {
            $Path = "$($RouteGroup.Path)$($Path)"
        }

So we can override the group value

Reply:
That would break how the logic works, as the idea is that you can have a group with a base `-Path` like `/api`, and then all child routes inherit this, like `/api/users` or `/api/groups` - so you only need to supply `/api` once at the RouteGroup level, and not again.

Reply:
For -Path I agree, but the other parameters can be convenient to have the option to override them